packages/runtime/src/useCases/consumption/attributes/DeleteRepositoryAttribute.ts

```diff
@@ -0,0 +1,70 @@
+import { Result } from "@js-soft/ts-utils";
+import { AttributesController, LocalAttribute } from "@nmshd/consumption";
+import { AccountController, CoreId } from "@nmshd/transport";
+import { Inject } from "typescript-ioc";
+import { AttributeIdString, RuntimeErrors, SchemaRepository, SchemaValidator, UseCase } from "../../common";
+
+export interface DeleteRepositoryAttributeRequest {
+    attributeId: AttributeIdString;
+}
+
+class Validator extends SchemaValidator<DeleteRepositoryAttributeRequest> {
+    public constructor(@Inject schemaRepository: SchemaRepository) {
+        super(schemaRepository.getSchema("DeleteRepositoryAttributeRequest"));
+    }
+}
+
+export class DeleteRepositoryAttributeUseCase extends UseCase<DeleteRepositoryAttributeRequest, void> {
```

Comment:
this usecase has a massive amount of logic, I think most could and should be extracted to the AttributesController

Reply:
<img src="https://media3.giphy.com/media/DEnaNFg7szMO7TwNYZ/giphy.gif"/>